ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_control_flow_operators.py

```diff
@@ -113,3 +113,41 @@ def _test_branch_2(x):
         branches=[_test_branch_1, _test_branch_2],
         operand=x[0],
     )
+
+
+@handle_frontend_test(
+    fn_tree="jax.lax.fori_loop",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        min_num_dims=1,
+        min_dim_size=1,
+    ),
+    lower=st.integers(min_value=-10, max_value=10),
```

Comment:
The range of lower and upper variables is limited to prevent looping for an excessively large number of iterations. The range includes positive and negative values, as well as the possibility of upper being less than lower.